login.html

```diff
@@ -0,0 +1,110 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>PetPal - Log In</title>
+
+  <!-- Import Fonts and Icons -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.css">
+
+  <!-- Custom Styles -->
+  <link rel="stylesheet" href="css/base.css">
+  <link rel="stylesheet" href="css/global.css">
+  <link rel="stylesheet" href="css/login-signup.css">
+</head>
+
+<body>
+  <div class="wrapper">
+    <header>
+      <a href="/" class="logo">
+        <img src="/img/logo_ref.png">
+        <span>PetPal</span>
+      </a>
+      <nav class="header__nav">
+        <div class="nav__menu">
+          <label for="nav__menu-toggle"><div class="nav__hamburger"></div></label>
+          <input type="checkbox" name="nav__menu-toggle" id="nav__menu-toggle">          
+          <ul>
+            <li><a href="#" class="nav__link">About Us</a></li>
+            <li><a href="#" class="nav__link">Adopt</a></li>
+            <li><a href="#" class="nav__link">Shelters</a></li>
+            <li><a href="#" class="nav__link">Donate</a></li>
+          </ul>
+        </div>
+        <div class="nav__icons">
+          <i class="material-icons">search</i>
+          <i class="material-icons">notifications</i>
+          <i class="material-icons">account_circle</i>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <div class="main__wrapper">
+        <div class="login__menu" >
+          <button class="login__menu-type active" id="seeker" onclick="seeker()">
+            <h6>Seeker</h6>
+          </button>
+          <button class="login__menu-type" id="shelter" onclick="shelter()">
+            <h6>Shelter</h6>
+          </button>
+          <script>
+            var se = document.getElementById("seeker");
+            var sh = document.getElementById("shelter");
+    
+            function seeker(){
+              se.classList.add("active");
+              sh.classList.remove("active");
+            }
+            function shelter(){
+              se.classList.remove("active");
+              sh.classList.add("active");
+            }
+          </script>
+        </div>
+        <div class="login__wrapper">
+          <h4 class="login__form-title">Log In</h4>
+          <form action="javascript:void(0);" method="post" class="login__form">
+            <input type="email" name="email" id="login__email" placeholder="Username/Email" required>
+            <input type="password" name="password" id="login_pwd" placeholder="Password" required>
+            
+            <!-- Use submit button instead of a tag once form becomes non-static -->
```

Comment:
Refer to comment in signup.html

Reply:
Fixed in [force-push](https://github.com/JuneOh710/PetPal/compare/ed734821be957cb2b6c8cdca82520d86c44a8974..1da33d4db46bc811b42d34b6387a0acad9354278)